tests/test_lookml.py

```diff
@@ -1137,6 +1155,7 @@ def test_lookml_actual(mock_glean_ping, runner, glean_apps, tmp_path, msg_glean_
             "explores": [
                 {
                     "name": "baseline",
+                    "description": "Explore for the baseline ping. The baseline ping is foo.",
```

Comment:
I didn't test the actual output, but this seems to indicate it should work.

Reply:
Okay, we should test this inside Looker. Run the following:
```
export HUB_BRANCH_PUBLISH="glean-ping-descriptions"
export SPOKE_BRANCH_PUBLISH="glean-ping-descriptions"
export GIT_SSH_KEY_BASE64=$(cat ~/.ssh/id_rsa | base64)
make build && make run
```

And it should push to `glean-ping-descriptions` in looker-hub and `glean-ping-descriptions-working` in looker-spoke-default. You can check those two branches out in Looker and ensure that the docs look correct and the LookML isn't broken.

Reply:
Things look pretty good in `looker-hub`: https://github.com/mozilla/looker-hub/blob/glean-ping-descriptions/fenix/explores/baseline.explore.lkml

I'm not sure what to look for in `looker-spoke-default`: https://github.com/mozilla/looker-spoke-default/tree/glean-ping-descriptions-working